packages/meditrak-server/src/routes/GETAnswers/GETAnswers.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { GETHandler } from '../GETHandler';
+import { assertAnswerPermissions, createAnswerDBFilter } from './assertAnswerPermissions';
+import { assertSurveyResponsePermissions } from '../GETSurveyResponses';
+import { allowNoPermissions, assertAnyPermissions, assertBESAdminAccess } from '../../permissions';
+
+/**
+ * Handles endpoints:
+ * - /answer
+ * - /answers/id
+ * - /surveyReponses/id/answers
+ */
+
+export class GETAnswers extends GETHandler {
+  assertUserHasAccess() {
+    return this.assertPermissions(allowNoPermissions);
```

Comment:
This is probably in enough places now that it's not worth having `assertUserHasAccess` as a required function. What do you think? Seems like we might be nulling it out more often than we're using it.

There's another check on the request to make sure permissions are checked at some point anyway right?